tests/components/Microsoft.DotNet.UpgradeAssistant.Tests/UpdateContextPropertiesTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Autofac;
+using Autofac.Extras.Moq;
+using Microsoft.Extensions.Logging.Abstractions;
+using Xunit;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Tests
+{
+    public class UpdateContextPropertiesTests
+    {
+        [Fact]
+        public async Task GetAndSetWork()
```

Comment:
You're getting some build warnings (errors as warnings are turned on) because this should be void.

Reply:
Argh, my bad. Would you consider turning off documentation requirements in the project if they are not followed? The warnings for this were hidden by over 200 documentation warnings.

Reply:
There's some sort of weirdness going on with VS and our editorconfig. I see this same thing (real warnings are drowned out by documentation warnings). The thing that's weird about it is that [we do have documentation warnings disabled](https://github.com/dotnet/upgrade-assistant/blob/main/.editorconfig#L262)!

What I usually do is build from the command line (msbuild /t:rebuild) before pushing commits because the documentation warnings don't show up when built outside of VS.

Reply:
I've noticed this as well but thought it was a preview thing.

It's probably due to editor config vs ruleset. We may want to just go back to rulesets.....

Reply:
@mjrousos @twsouthwick I took a peek at editorconfig and to me it seems that rules SA1600 and SA1601 are accidentally written as CA1600 and CA1601.

Reply:
Good catch, @oteinone! I can't believe I missed that! We've run into issues with preview VS's handling of .editorconfig before, so I figured this was just another instance of that type of problem that would need investigated at some point. I'll get a PR opened; it'll be nice to not have all those warnings!